components/patient-data/impl/src/test/java/org/phenotips/data/internal/controller/GeneListControllerTest.java

```diff
@@ -118,8 +119,8 @@
 
     private static final String JSON_OLD_CANDIDATE_GENE_KEY = "candidate";
 
-    private static final List<String> STATUS_VALUES = Arrays.asList("candidate", "rejected", "rejected_candidate",
-        "solved", "carrier");
+    private static final List<String> STATUS_VALUES = Arrays.asList("candidate", "rejected", "rejectedcandidate",
+        "solved", "carrier", "candidate_novel_disease", "candidate_novel_phen", "umc", "umc_vus", "umc_msv");
```

Comment:
These values should be extracted from the XClass instead of being duplicated here